server/src/main/java/org/elasticsearch/index/mapper/GeoPointFieldMapper.java

```diff
@@ -478,6 +482,17 @@ public IndexFieldData.Builder fielddataBuilder(FieldDataContext fieldDataContext
             throw new IllegalStateException("unknown field data type [" + operation.name() + "]");
         }
 
+        @Override
+        public BlockLoader blockLoader(BlockLoaderContext blContext) {
+            if (hasDocValues()) {
+                return new BlockDocValuesReader.LongsBlockLoader(name());
+            }
+            // TODO: Currently we use longs in the compute engine and render to WKT in ESQL
+            return new BlockSourceReader.LongsBlockLoader(
+                valueFetcher(blContext.sourcePaths(name()), nullValue, GeometryFormatterFactory.WKT)
+            );
+        }
```

Comment:
Right right - you'll build WKT when you load from _source. You just aren't doing that. I suppose my question was "is the WKT thing here a parser config or can you still parse all supported point types?" Because I'm ok delaying any questions about parsing from source into the lossless render until later.